src/platform/i386/kernel.c

```diff
@@ -135,9 +137,9 @@ kmain(struct multiboot *mboot, u32_t mboot_magic, u32_t esp)
 #define MAX(X, Y) ((X) > (Y) ? (X) : (Y))
 	unsigned long max;
 
-	tss_init();
-	gdt_init();
-	idt_init();
+	tss_init(0);
```

Comment:
There is a `#define INIT_CORE 0` in cos_config.h, might want to use that.

Reply:
good point